### PR TITLE
Add docs-src. This is where the manually created docs will live

### PR DESCRIPTION
**Pull request by gkoz**
This directory will not be visible on the site. Instead the rendered docs
(_site/docs-src) will be (manually at first) copied to the docs repo to
appear at http://gtk-rs.org/docs keeping a nice url hierarchy.

### CODE REVIEW

**Comment by gkoz** [comment]
I've actually pushed the rendered version to the `docs` repo as a preview: http://gtk-rs.org/docs/

**Comment by GuillaumeGomez** [comment]
I don't like the new style at all. It's way too white. The background of the divs looks like pink. Well, I just can't stand it.

**Comment by gkoz** [comment]
Just noticed the broken shell snippets highlighting.

**Comment by gkoz** [comment]
New style? I haven't touched any styles at all.

**Comment by GuillaumeGomez** [comment]
screenshot from 2016-01-27 17 21 36

That's what I see. :-/

**Comment by gkoz** [comment]
That's how the rest of the site is styled, (except for the rustdoc of course).

**Comment by GuillaumeGomez** [comment]
Seriously? Hum... Before any more update, I'm gonna change the colors, make them less lighty.

**Comment by gkoz** [comment]
Fixed highlighting in the PR (not in the preview).

**Comment by gkoz** [comment]
Note that this change is completely orthogonal to any style issues.

**Comment by GuillaumeGomez** [comment]
Can you update the preview as well please?

**Comment by gkoz** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
Still not good. Don't bother, I'll do something about the colors.

**Comment by gkoz** [comment]
Of course I still didn't touch any colors because this PR isn't about colors.

**Comment by gkoz** [comment]
Added readme. This should be it.

**Comment by GuillaumeGomez** [comment]
Perfect, thanks!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# The gtk-rs.org site
+
+The site is hosted on [GitHub Pages](https://pages.github.com/) powered by
+[Jekyll](http://jekyllrb.com/).
+
+When authoring any changes you can try them out locally by running (requires ruby)
```
Comment (GuillaumeGomez) [inline README.md]: Missing ':' I think at the end of the sentence.

**Comment by gkoz** [comment]
Updated

**Pull request merged**